articles/quickstart/spa/_includes/_login_preamble.md

```diff
@@ -1,3 +1,8 @@
-The first step in adding authentication to your ${library} application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the [hosted login page](https://auth0.com/docs/hosted-pages/login).
+Use the  hosted login page to provide a way for your users to log in to your ${library} application.
```

Comment:
As discussed on slack, this is a placeholder that is replaced when the snippet is inserted into an article (the variable is defined in the insert command).